src/collinearity_analysis.py

```diff
@@ -0,0 +1,63 @@
+"""
+This script make collinearity analysis based on 4 predictors for our linear regression
+"""
+
+import pandas as pd
+from scipy.stats import pearsonr
+import matplotlib.pyplot as plt
+import seaborn as sns
+import fire
+
+def collinearity_analysis(dataset_path:str,barchart_name:str,heatmap_name:str):
```

Comment:
please format using [black formatter](https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter). You can do this by installing this vscode extension and right click then choosing "format document".

Reply:
really helpful,thx!